src/Adapter/Product/Image/Update/ProductImageUpdater.php

```diff
@@ -170,6 +170,9 @@ public function updatePosition(Image $image, int $newPosition): void
         ];
 
         try {
+            //@todo: Does this have a related tests at all?
+            // needs additional testing. It insert productId 0 and throws CannotUpdateProductImageException,
+            // because it seems to not load the image fully. id_product and id_image are null.
```

Comment:
Is this still an issue? There was a bug previously because the ImageCore constructor didn't handle the shopId parameter properly, it should work now but the used shopId must be an associated one or it will fail

If there's something unstable here we should dig into it, the position update was behat tested in single shop but seems like it's not in multishop we should probably add related tests This could be handled in another PR though (then we should open an issue)

Reply:
hmm this was fixed, I can get rid of it